util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java

```diff
@@ -125,6 +125,7 @@ public void addEventHandlerWithResyncPeriod(
           log.warn(
               "DefaultSharedIndexInformer#resyncPeriod {} is smaller than resyncCheckPeriod {} and the informer has already started. Changing it to {}",
               resyncPeriodMillis,
+              resyncCheckPeriodMillis,
```

Comment:
why is this change needed?

Reply:
If you look at the log message- _"DefaultSharedIndexInformer#resyncPeriod {} is smaller than resyncCheckPeriod {} and the informer has already started. Changing it to {}"_. It has 3 placeholders but we are supplying only 2 variables to log.

Reply:
I can remove this if this looks redundant.

Reply:
No that's fine, I was just confused about how adding a parameter was necessary if it was compiling before, but now I realize that it's a varargs function.